lorien/InfiniteCanvas/Cursor/SelectCursor/SelectCursorInfo.tscn

```diff
@@ -0,0 +1,40 @@
+[gd_scene load_steps=3 format=2]
```

Comment:
Actually this scene shuold be removed. I was using it to show some values in a label following the SelectCursor (f.i. size of the selection area, selected lines, etc..), but then I tought that it would only add something that most of the users are not interested seeing, and I ended up not using it to keep the UI simple.